third-party/patches/fbthrift-2018-08-20.patch

```diff
@@ -0,0 +1,192 @@
+diff --git a/thrift/compiler/parse/thriftl.ll b/thrift/compiler/parse/thriftl.ll
```

Comment:
Why we need the changes?  The bison and flex we used require that?

Reply:
We will use the bison/flex installed in the system if the versions meet our need. But if they are too new, there will be problems. Actually, this is a backport from the latest fbthrift.